querier.go

```diff
@@ -0,0 +1,28 @@
+package sqlr
+
+import (
+	"context"
+	"database/sql"
+)
+
+// RowQuerier requests a row in specific context
```

Comment:
```suggestion
// RowQuerier requests a row in a specific context
```
Executer has already this form so I suggest moving to it for the rest